remodel/monkey.py

```diff
@@ -1,7 +1,7 @@
 import rethinkdb as r
 
-import connection
-
+import remodel.connection
+import rethinkdb.ast
```

Comment:
You haven't used this import here. Furthermore, `rethinkdb` is already imported 3 lines above.